wren-ui/src/apollo/server/resolvers/askingResolver.ts

```diff
@@ -398,6 +403,32 @@ export class AskingResolver {
     });
   }
 
+  public async generateThreadResponseChart(
+    _root: any,
+    args: { threadId: number; responseId: number },
+    ctx: IContext,
+  ): Promise<ThreadResponse> {
+    const project = await ctx.projectService.getCurrentProject();
+    const { threadId, responseId } = args;
```

Comment:
Could you kindly explain why we need to pass both threadId and threadResponseId from the client?
I think threadResponseId is enough to check and fetch the info you need